### PR TITLE
Config with zope product

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code. (Maybe the isort config still needs some changes.)
* Support Python 3.9
* Drop leftovers from supporting Zope < 2.8.

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -1,7 +1,6 @@
 [buildout]
 extends =
-    https://raw.githubusercontent.com/zopefoundation/Zope/master/versions.cfg
-    https://raw.githubusercontent.com/zopefoundation/Zope/master/sources.cfg
+    http://zopefoundation.github.io/Zope/releases/master/versions.cfg
 develop = .
 parts = interpreter test coverage
```
Comment (dataflake) [inline buildout.cfg]: remove `coverage` here

File: MANIFEST.in [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
 include *.rst
 include *.txt
+include buildout.cfg
+include tox.ini
 
-recursive-include src *
-
-global-exclude *.pyc
-global-exclude *.pyo
+recursive-include src *.pt
```
Comment (dataflake) [inline MANIFEST.in]: There's so much stuff here by default, are you working off an older version of the `meta` package?

File: setup.py [hunk 3]
```diff
@@ -10,21 +12,23 @@
...
         'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: Implementation :: CPython',
     ],
     keywords='version control',
     license='ZPL',
     author='Zope Foundation and Contributors',
     author_email='zope-dev@zope.org',
-    url='https://pypi.org/project/Products.ZopeVersionControl/',
+    url='https://github.com/zopefoundation/Products.ZopeVersionControl',
```
Comment (dataflake) [inline setup.py:31]: Would you add a `project_urls` dictionary as well as `python_requires`?

File: src/Products/ZopeVersionControl/EventLog.py [hunk 4]
```diff
@@ -31,6 +33,7 @@ def __init__(self):
     security = ClassSecurityInfo()
 
     security.declarePrivate('addEntry')
```
Comment (dataflake) [inline src/Products/ZopeVersionControl/EventLog.py]: When dropping Zope 2 support you could convert this to decorators

File: src/Products/ZopeVersionControl/nonversioned.py [hunk 5]
```diff
@@ -15,16 +15,18 @@
 $Id$
```
Comment (dataflake) [inline src/Products/ZopeVersionControl/nonversioned.py]: Remove CVS artifact

File: src/Products/ZopeVersionControl/tests/__init__.py [hunk 6]
```diff
@@ -11,4 +11,4 @@
 #
 ##############################################################################
 
-__version__='$Revision: 1.2 $'[11:-2]
+__version__ = '$Revision: 1.2 $'[11:-2]
```
Comment (dataflake) [inline src/Products/ZopeVersionControl/tests/__init__.py]: I'd remove this CVS artifact

**Review by dataflake — CHANGES_REQUESTED** [review]

**Review by dataflake — APPROVED** [review]

File: MANIFEST.in (hunk 2 above)
Comment (icemac) [reply MANIFEST.in]: I was wondering, too so I checked whether I am on the current tip of `master`.
The after checking the `meta` repository here on GitHub just now I found out that https://github.com/zopefoundation/meta/pull/86 is not merged yet.

**Comment by icemac** [comment]
@dataflake Thank you for reviewing and improving this PR.

**Pull request merged**